src/server/system_services/pool_controllers.js

```diff
@@ -78,78 +73,34 @@ class ManagedStatefulSetPoolController extends PoolController {
         if (delta === 0) return;
```

Comment:
I think you should not check the number of pod but read the status on the pvpool backing store in order to get the number to preform the scale.

Reply:
fixed